tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "build",
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "strict": true,
+    "jsx": "preserve",
+    "baseUrl": ".",
+    "skipLibCheck": true,
+    "paths": {
+      "@components/*": [".vitepress/theme/components/*"],
+      "@constants/*": [".vitepress/theme/constants/*"]
+    }
+  },
+  "include": ["src/**/*", ".vitepress/**/*"],
+  "exclude": ["node_modules", "build"]
```

Comment:
I guess `cache` in `.vitepress` directory should be excluded, too?

Reply:
Oh, yes!!

Reply:
Then I will handle this in another PR!

Reply:
Thank you, that’s very helpful!

Reply:
See #32